src/components/textField/index.tsx

```diff
@@ -107,6 +109,8 @@ const TextField = (props: InternalTextFieldProps) => {
     }
   }, [leadingAccessory]);
 
+  const _helperText = <Text $textNeutralHeavy subtext>{helperText}</Text>;
```

Comment:
Why not render it directly without the const, it's not necessarily more readable like this, it's a single line anyway and it also kinda weird you create a JSX code before all the hooks and the props destruction

Reply:
Done